### DISCUSSION TITLE
Allow for Review Scores / User Rating Metadata

### DISCUSSION THREAD

**Discussion by kanjieater** [gotson/komga#1531] [Ideas]
### Describe your suggested feature

Being able to see the score of a series would be useful metadata in deciding what to read in your library. Metadata aggregators like komf already support this via tags (though the score tags can get buried behind if you have many tags)

Doing something that allows for visuals like 5 stars with the actual number out of 5.0 would be nice.
![image](https://github.com/gotson/komga/assets/32607317/3c30c950-bba3-447e-addc-04d8036aac73)

I think there are a lot of different area's it could go in here, so whatever gotson prefers:
![image](https://github.com/gotson/komga/assets/32607317/14ed5dac-21a9-464f-85e6-69cef18f965a)

### Other details

Down the road this could also be used for sorting the library by review scores or even searching by review scores. I think it's more critical to allow for the data as an independent meta field for now though.

It may make sense to allow for multiple scores, similar to links (anilist gives it a 4, but mangaupdates gives it a 3). Then something like Komf could aggregate scores and create something like an average score in addition: Komf: 3.5 from (4+3/2)

This is for external sites' scores. I think there could also be a field for user's personal scores - where users from the library could rate the works themselves - but to be clear that's not what this feature request is. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by bulletproof2k** [comment]
Something similar was implemented in [this project](https://github.com/wrecks-code/komga-sync-comic-ratings), but unfortunately the userscript does not work correctly. Some elements are not displayed at all or are displayed only in dark mode.

**Comment by G1ebel** [comment]
I just posted a little something I did to fix the missing rating system. Take a look if you are interested: https://github.com/gotson/komga/discussions/2401
